packages/broker/test/unit/config.test.ts

```diff
@@ -23,61 +23,43 @@ describe('overrideConfigToEnvVarsIfGiven', () => {
             }
         }
         process.env.STREAMR__BROKER__CLIENT__AUTH__PRIVATE_KEY = '0x111'
-        process.env.STREAMR__BROKER__CLIENT__NETWORK__TRACKERS_1__ID = 'tracker1-id'
-        process.env.STREAMR__BROKER__CLIENT__NETWORK__TRACKERS_1__HTTP = 'tracker1-http'
-        process.env.STREAMR__BROKER__CLIENT__NETWORK__TRACKERS_2__ID = 'tracker2-id'
-        process.env.STREAMR__BROKER__CLIENT__NETWORK__TRACKERS_2__HTTP = 'tracker2-http'
-        process.env.STREAMR__BROKER__CLIENT__NETWORK__TRACKER_PING_INTERVAL = '-0.5'
         process.env.STREAMR__BROKER__CLIENT__ORDER_MESSAGES = 'true'
         process.env.STREAMR__BROKER__CLIENT__GAP_FILL = 'false'
+        // process.env.STREAMR__BROKER__CLIENT_NETWORK__LAYER0__PEER_DESCRIPTOR__KADEMLIA_ID = 'kademliaID'
```

Comment:
The environment variable cannot parse the layer0 object. Rename to ie. controlLayer